### PR TITLE
Moved jss-default-unit

### PR DESCRIPTION
**Pull request by HenriBeck**
Signed-off-by: Henri Beck <henribeck.dev@gmail.com>

### CODE REVIEW

**Comment by HenriBeck** [comment]
Any suggestions for a name change? @kof

**Comment by kof** [comment]
I think its also syntactic, so jss-plugin-syntax-default-unit

File: packages/jss-plugin-syntax-expand/.size-snapshot.json [hunk 1]
```diff
@@ -8,5 +8,15 @@
     "bundled": 11240,
     "minified": 4432,
     "gzipped": 1544
+  },
+  "dist/jss-plugin-syntax-expand.js": {
```
Comment (kof) [inline packages/jss-plugin-syntax-expand/.size-snapshot.json]: wrong name

Comment (kof) [reply packages/jss-plugin-syntax-expand/.size-snapshot.json]: ah its a change from a different package here

Comment (HenriBeck) [reply packages/jss-plugin-syntax-expand/.size-snapshot.json]: Yeah, forgot to build and it created them only now with the correct names.

**Pull request merged**